polylabel.js

```diff
@@ -25,7 +25,11 @@ function polylabel(polygon, precision, debug) {
     var cellSize = Math.min(width, height);
     var h = cellSize / 2;
 
-    if (cellSize === 0) return [minX, minY];
+    if (cellSize === 0) {
+        var degeneratePoleOfInaccessibility = [minX, minY];
```

Comment:
Are `minX` and `minY` guaranteed to be zero in this condition? If so, we could further simplify.